jbpm-wb-integration/jbpm-wb-integration-client/src/test/java/org/jbpm/workbench/wi/client/workitem/ServiceTaskUploadFormPresenterTest.java

```diff
@@ -62,7 +65,10 @@
 
     @Before
     public void before() {
-        when(serviceTaskService.addServiceTasks(any())).thenReturn(new ArrayList<>());
+        Map<String, List<String>> resultMap = new HashMap<String, List<String>>();
```

Comment:
please also add test covering the actual contents of notification event (what was created vs. what was updated) ... a variant of testUploadSuccessful() test.

Reply:
done